shared/surface_flux.F90

```diff
@@ -272,7 +281,7 @@ subroutine surface_flux_1d (                                           &
        q_atm,    q_surf0,  dw_atmdu,  dw_atmdv,  w_gust,   &
        zu,       zt,       zq
 
-  integer :: i, nbad
+  integer :: i, j, nbad
```

Comment:
Please remove the _j_ declaration since the creation of the _iter_monin_obukhov_ocean_ function removed the need for it here.